cime_config/machines/config_machines.xml

```diff
@@ -55,6 +55,77 @@
      This allows using a different mpirun command to launch unit tests
 
   -->
+  <machine MACH="spock">
+    <DESC>Spock. NCCS moderate-security system that contains similar hardware and software as the upcoming Frontier system at ORNL.</DESC>
+    <NODENAME_REGEX>.*spock.*</NODENAME_REGEX>
+    <OS>CNL</OS>
+    <COMPILERS>gnu</COMPILERS>
+    <MPILIBS>mpich</MPILIBS>
+    <PROJECT>cli133</PROJECT>
+    <CIME_OUTPUT_ROOT>/gpfs/alpine/$PROJECT/scratch/$ENV{USER}/e3sm_scratch</CIME_OUTPUT_ROOT>
```

Comment:
To avoid permission issues, typical location is `proj-shared`:
```
    <CIME_OUTPUT_ROOT>/gpfs/alpine/$PROJECT/proj-shared/$ENV{USER}/e3sm_scratch/spock</CIME_OUTPUT_ROOT>
```